sw/device/silicon_creator/rom_ext/e2e/attestation/print_certs.c

```diff
@@ -0,0 +1,82 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/base/status.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/runtime/print.h"
+#include "sw/device/lib/testing/test_framework/ottf_main.h"
+#include "sw/device/silicon_creator/lib/drivers/flash_ctrl.h"
+
+OTTF_DEFINE_TEST_CONFIG();
+
+const char base64[] =
+    "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
+
+void base64_encode(char *dest, const uint8_t *data, int32_t len) {
+  *dest = '\0';
+  for (int32_t i = 0; len > 0; i += 3, len -= 3) {
+    // clang-format off
+    uint32_t val = (uint32_t)(data[i] << 16 |
+                              (len > 1 ? data[i + 1] << 8 : 0) |
+                              (len > 2 ? data[i + 2] : 0));
+    // clang-format on
+    *dest++ = base64[(val >> 18) & 0x3f];
+    *dest++ = base64[(val >> 12) & 0x3f];
+    *dest++ = len > 1 ? base64[(val >> 6) & 0x3f] : '=';
+    *dest++ = len > 2 ? base64[(val >> 0) & 0x3f] : '=';
+    *dest = '\0';
+  }
+}
+
+status_t print_cert(char *dest, const flash_ctrl_info_page_t *info_page) {
+  uint8_t data[2048];
+  TRY(flash_ctrl_info_read(info_page, 0, sizeof(data) / sizeof(uint32_t),
+                           data));
+
+  int16_t len = (int16_t)(data[2] << 8 | data[3]);
```

Comment:
use this function instead: https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/lib/cert/cert.c;l=18 ?

Reply:
That function doesn't exist in `earlgrey_es_sival`.  I added a copy of it to the test program with a note to remove it when merging to master.